mod_ci/controllers.py

```diff
@@ -905,6 +907,34 @@ def blocked_users():
             g.db.add(blocked_user)
             g.db.commit()
             flash('User blocked successfully.')
+            try:
+                # Remove any queued pull request from blocked user
+                gh = GitHub(access_token=g.github['bot_token'])
+                repository = gh.repos(g.github['repository_owner'])(g.github['repository'])
+                # Getting all pull requests by blocked user on the repo
+                pulls = gh.repos('ccextractor')('sample-platform').pulls.get()
```

Comment:
Why do this if you have a `repository` object above that contains it all up to the `.pulls.get()` part?